src/accessories/TokenArranger.ttslua

```diff
@@ -244,17 +284,70 @@ function layout(_, _, isRightClick)
   local pos           = self.getPosition() + Vector(3.55, -0.05, -3.95)
   local location      = { x = pos.x, y = pos.y, z = pos.z }
   local current_value = data[1].value
+  local token_count = 0
+	local value_count = 1
+	local cumulative_percentage = 0
```

Comment:
Here and following: Make sure the indentation is consistent and aligns with the existing code.

Reply:
Went through and did a check, I believe it should be consistent now